tsconfig.json

```diff
@@ -4,4 +4,4 @@
       { "path": "./tsconfig.app.json" },
       { "path": "./protocol/tsconfig.json" }
   ]
-}
\ No newline at end of file
+}
```

Comment:
EOL warning

Reply:
same as the other - I think this is in the negative so shouldn't be in anymore?